### PR TITLE
Created Java's ArrayList Get() method term entry

### LINKED ISSUE #1441: [Term Entry] Java ArrayList: `.get()`
### Reason/inspiration (optional)

We'd like to add a new term entry for the `.get()` method under the Java: ArrayList concept. In the new entry, please include the following:

- A "Syntax" section with a \```pseudo code snippet.
- An "Example" section with a \```java code snippet.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by hectorconte**
### Description

On this contribution, I have created the term entry for Java's ArrayList Get() method, following guidelines and giving an example.

Closes #1441 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/array-list/terms/get/get.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+Title: '.get()'
+Description: 'Brings the element present in a specified position in an ArrayList.'
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:3]:
```suggestion
Retrieves the element present in a specified position in an ArrayList
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.get()`** method brings the element at a specified position within an `ArrayList` instance.
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:15]:
```suggestion
The **`.get()`** method retrieves the element at some index in an `ArrayList`.
```

Comment (HishamT) [reply content/java/concepts/array-list/terms/get/get.md:15]: Add a mention to what `.get()` returns if the element is not found.

File: content/java/concepts/array-list/terms/get/get.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    ArrayList<String> studentList = new ArrayList<String>();
+
+    // Add new values to the studentList
+    studentList.add("John");
+    studentList.add("Lily");
+    studentList.add("Samantha");
+    studentList.add("Tony");
+
+    // `.get()` method stores element's value
+    String StudentOne = studentList.set(0);
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:44]:
```suggestion
    String StudentOne = studentList.get(0);
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+The **`.get()`** method brings the element at a specified position within an `ArrayList` instance.
+
+## Syntax
+
+```pseudo
+arrayListInstance.get(index);
+```
+
+An element at a specified `index` is brought in an `arrayListInstance`.
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:23]:
```suggestion
The element at index `index` in `arrayListInstance` is returned.
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## Example
+
+In the example below, an empty `ArrayList` instance `studentList` is created and can hold `String`-type elements. Next, a few elements are added with the [`.add()`](https://www.codecademy.com/resources/docs/java/array-list/add) method. Lastly, two students are brought up with the `.get()` method:
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:27]:
```suggestion
In the example below, an empty `ArrayList`, `studentList`, is created containing a few elements of type `String`. The `.get()` method retrieves the students at indexes 0 and 1.
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+    // Add new values to the studentList
+    studentList.add("John");
+    studentList.add("Lily");
+    studentList.add("Samantha");
+    studentList.add("Tony");
+
+    // `.get()` method stores element's value
+    String StudentOne = studentList.set(0);
+    String StudentTwo = studentList.set(1);
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:45]:
```suggestion
    String StudentTwo = studentList.get(1);
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+    // Output ArrayList and searched elements
+    System.out.println("ArrayList: " + studentList);
+    System.out.println("Searched Elements: " + StudentOne + " and " + StudentTwo);
+  }
+}
+```
+
+The output should look like this:
```
Comment (HishamT) [inline content/java/concepts/array-list/terms/get/get.md:54]:
```suggestion
The output will look like this:
```

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @hectorconte, I have finished the review of your entry and left you some comments.

**Comment by hectorconte** [comment]
Hi @HishamT, I will correct this ASAP, thanks for checking my work!

**Review by hectorconte — COMMENTED** [review]
All done! Will wait next review. Thanks again!

**Review by HishamT — APPROVED** [review]
Hey @hectorconte, I have made some minor changes. This looks good.

**Comment by hectorconte** [comment]
@HishamT Thanks for reviewing it!

File: content/java/concepts/array-list/terms/get/get.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+The **`.get()`** method retrieves the element at some index in an `ArrayList`.
+
+## Syntax
+
+```pseudo
+arrayListInstance.get(index);
+```
+
+The element at index `index` is `arrayListInstance` is returned. If the index is out of range, an `IndexOutOfBoundsException` will be thrown.
```
Comment (caupolicandiaz) [inline content/java/concepts/array-list/terms/get/get.md:23]:
```suggestion
The element at index `index` in `arrayListInstance` is returned. If the index is out of range, an `IndexOutOfBoundsException` error will be thrown.
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+## Example
+
+In the example below, an empty `ArrayList`, `studentList`, is created containing a few elements of type `String`. The `.get()` method retrieves the students at indexes 0 and 1.
```
Comment (caupolicandiaz) [inline content/java/concepts/array-list/terms/get/get.md:27]:
```suggestion
In the example below, an empty `ArrayList`, `studentList`, is created and populated with several elements of type `String`. The `.get()` method retrieves the students at indexes 0 and 1.
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+```java
+import java.util.ArrayList;
```
Comment (caupolicandiaz) [inline content/java/concepts/array-list/terms/get/get.md:30]:
```suggestion
// Students.java
import java.util.ArrayList;
```

File: content/java/concepts/array-list/terms/get/get.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+    ArrayList<String> studentList = new ArrayList<>();
+
+    // Add new values to the studentList
+    studentList.add("John");
+    studentList.add("Lily");
+    studentList.add("Samantha");
+    studentList.add("Tony");
+
+    // `.get()` method stores element's value
```
Comment (caupolicandiaz) [inline content/java/concepts/array-list/terms/get/get.md:43]:
```suggestion
    // `.get()` method extracts the element's value
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@hectorconte overall this looks pretty good, just made some minor edits for clarity

**Comment by yangc95** [comment]
@hectorconte checking in to see if you meant to close out this PR. I notice the repository is still on your GitHub profile. Let me know if you need any assistance or of your intentions. Thank you! 🙂

**Comment by hectorconte** [comment]
Hi @yangc95 ! Sorry, I closed this PR as a mistake, since I deleted (again, unintentionally) my forked repository which I re-forked afterwards. So I opened a new PR to keep working there. Sorry!

**Comment by yangc95** [comment]
@hectorconte gotcha! Thanks for letting me know ☺️ I see your PR here: https://github.com/Codecademy/docs/pull/1557

It would be a good idea to rename the PR similarly to how you did so here. In any case, if you have any further questions, reach out to any of us maintainers ^^

cc: @caupolicandiaz @HishamT

**Comment by hectorconte** [comment]
@yangc95 Yes! That's the one! Thanks a lot

**Pull request closed without merging**